tests/test_queryBuilder.py

```diff
@@ -59,34 +59,34 @@ def testDatasetOriginInfoDef(self):
         self.assertEqual(originInfo.getOutputCollection("ds2"), "out2")
 
     def testInstrumentDimensions(self):
-        """Test involving only Instrument dimensions, no joins to SkyMap"""
+        """Test involving only instrument dimensions, no joins to skymap"""
```

Comment:
SkyMap -> skymap and not sky_map seems anomalous but I have no problem with skymap.

Reply:
Yeah, it's a little inconsistent, but I think there's a bit of precedent for it and people will find this a bit less jarring.

(More generally, I recently came to the sad conclusion that even within pure camelCase, our brains just refuse to make the same word boundaries in different contexts - we almost always write `BBox` when the first "B" is capitalized, but *always write `bbox` instead of `bBox` when the first "b" is lowercase, for example.)

Reply:
and then we have the whole camelCase insistence that "LSST" become "Lsst"... 😄